terraform/Makefile

```
@@ -43,8 +43,12 @@ import_resources: rename_resources
 plan: import_resources
 	terraform plan
 
+.PHONY: delete_lambda_versions
+delete_lambda_versions:
+	python $(project_root)/scripts/delete_lambda_versions.py
+
 .PHONY: apply
-apply: import_resources
+apply: import_resources delete_lambda_versions
```

```suggestion
apply: import_resources delete_lambda_versions
```

This is saying that `apply` depends on `import_resources` and `delete_lambda_versions` but not in which order these latter two are executed. It is possible under certain [conditions](https://stackoverflow.com/questions/41006891/makefile-order-of-dependencies/77176584#77176584) that `make apply` will run `delete_lambda_versions` as the very first target, which could have disastrous consequences if, say, the current configuration was invalid.